content/en/lotus/manage/slasher-and-disputer.md

```diff
@@ -0,0 +1,84 @@
+lead: "Lotus Slasher and Lotus Disputer are essential services provided by the Filecoin protocol. They help maintain network integrity, prevent bad actors, and reward active participants."
+draft: true
+menu:
+    lotus:
+        parent: "lotus-management"
+aliases:
+    - /docs/set-up/chain-management/
+weight: 455
+toc: true
+---
+
+# The Lotus Slasher & Lotus Disputer
+
+Lotus Slasher and Lotus Disputer are essential services provided by the Filecoin protocol. They help maintain network integrity, prevent bad actors, and reward active participants. 
+
+The Slasher and Disputer services are designed to be highly efficient and lightweight processes that require minimal resources. Enabling these services will have no detrimental effect on the performance of your Lotus node.
+
+# The Lotus Slasher
+
+The Filecoin protocol enforces penalties for three types of consensus faults that can be committed by block producers. To detect and report these faults, any user running a Lotus node can leverage the Lotus Slasher service and invoke the `ReportConsensusFault` method on the respective miner actor. Detailed information about these three consensus faults can be found [here](https://github.com/filecoin-project/lotus/blob/dbbcf4b2ee9626796e23a096c66e67ff350810e4/chain/vm/fvm.go#L113-L132).
+
+When a consensus fault is successfully identified by the Lotus Slasher, the block producer responsible for the fault is subjected to a fine equivalent to 5 times the current epoch's block reward value. Additionally, they are ineligible to produce any further blocks for a full finality period of 900 epochs. Furthermore, the offending block producer is prohibited from pre-committing or recovering storage during the same finality period.
+
+The succesful Lotus Slasher operator receives a reward equal to 25% of the current epoch's block reward value.
+
+## Run the Slasher
+
+The Lotus Slasher is already packagaed with Lotus so no additional installation steps are required. You **do not** need to be running `lotus-miner` to operate a Slasher.
+
+The Lotus Slasher functionality is activated by simply providing additional flags to the `lotus daemon` launch command as shown below.
+
+```shell
+lotus daemon --slash-consensus=true --slashdb-dir="/your/path"
+```
+
+The `--slash-consensus` and `--slashdb-dir` flags are both required to succesfully invoke the Slasher service.
+
+The `--slasher-sender` flag is also provided to optionally set the account to report consensus faults from.
+
+
+# The Lotus Disputer
+
+Prior to the introduction of [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md), Storage Providers were required to verify and record proofs of continued data storage in the form of on-chain WindowPoST messages. This approach often led to high costs and bandwidth usage associated with verifying proofs, especially during periods of congestion and rising gas fees. 
```

Comment:
This is just opinion, but I don't think this section needs to be written with so much history -- the motivation behind FIP-0010 doesn't seem too relevant here.

I'd consider rewriting this to just say that:
- WindowPoSts are important proofs submitted daily to ensure continued storage of all data on Filecoin
- These proofs are NOT actually validated on chain, in order to save chain bandwidth
- Instead, the network relies on off-chain validators to actually check the correctness of these proofs, and `Dispute` them if appropriate

And then everything you have from `Third-party Lotus node operators now...` onwards looks good!